File: Zerojudge-a518.cpp
// Author : ysh
// 06/20/2022 Mon 11:54:46.64
#include<bits/stdc++.h>
using namespace std;
int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);

    int a,b;
    while(cin>>a>>b) {
        if(a > b) swap(a,b);
        if(a == -1 && b == -1) {
            break;
        }
        cout<<min(b - a,(a + 100) - b)<<"\n";
    }
    return 0;
}

File: Zerojudge-a518.py
# Author : ysh
# 06/20/2022 Mon 11:51:42.21
while True:
    try:
        a = sorted([int(i) for i in input().split()])
        if(a[0] == -1 and a[1] == -1):
            break
        print(min(a[1] - a[0],(a[0] + 100) - a[1]))
    except:
        break
quit()